tasks/managed/apply-mapping/apply-mapping.yaml

```diff
@@ -22,22 +22,60 @@ spec:
       type: string
       description: Fail the task if the resulting snapshot contains 0 components
       default: "false"
+    - name: ociStorage
+      description: The OCI repository where the Trusted Artifacts are stored.
+      type: string
+    - name: ociArtifactExpiresAfter
+      description: Expiration date for the trusted artifacts created in the
+        OCI repository. An empty string means the artifacts do not expire.
+      type: string
+      default: "1d"
+    - name: sourceDataArtifact
+      type: string
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
   workspaces:
-    - name: config
+    - name: data
       description: The workspace where the extra config file containing the mapping and snapshot json reside
   results:
     - name: mapped
       type: string
       description: A true/false value depicting whether or not the snapshot was mapped.
+    - description: Produced trusted data artifact
+      name: sourceDataArtifact
+      type: string
   steps:
+    - name: skip-trusted-artifact-operations
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -eo pipefail
+
+        if [ "$(params.ociStorage)" == "empty" ]; then
```

Comment:
Should `empty` be the default for this parameter then?

Reply:
fixed